fronts-client/src/types/Collection.ts

```diff
@@ -189,10 +190,13 @@ type CollectionWithNestedArticles = CollectionFromResponse & {
 
 // previouslyCardIds is stored in a separate key to avoid losing ordering information during normalisation.
 interface Collection {
-	live?: string[];
 	previously?: string[];
 	previouslyCardIds?: string[]; // this contains ids for deleted articles on a collection
+	// uuids for draft groups
 	draft?: string[];
+	// uuids for live groups
+	live?: string[];
```

Comment:
I had initially thought that these were Card IDs, but in fact they are Group UUIDs! The hierarchy is Front -> Collection -> Group -> Card.